Dockerfile.dapper

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.12.1-alpine3.9
+FROM golang:1.14.8-alpine
```

Comment:
make go versions consistent, let's just go to 1.15, change in go mod file too

Reply:
@luthermonson done